tests/execute/local_subprocess/test_local_subprocess.py

```diff
@@ -77,7 +87,7 @@ def test_local_execute_basic_pass(  # noqa: PLR0913
     out_got, err_got = out_err.read_out_err()
     if show:
         assert out_got == out
-        expected = (f"{Fore.RED}{err}{Fore.RESET}" if color else err) if err else ""
+        expected = f"{Fore.__dict__[stderr_color]}{err}{Fore.RESET}" if color and err else err
```

Comment:
Use getattr instead of this dictionary lookup.

Reply:
Indeed, much nicer than the __dict__ approach.